com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/internal/ash/AshFrameHandler.java

```diff
@@ -23,6 +23,7 @@
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
This is never used. I guess maybe you were considering using ```AtomicBoolean``` instead? If you're not going to use this then please remove the import.